tests/mock_network/churn.rs

```diff
@@ -415,7 +431,7 @@ fn send_and_receive<R: Rng>(rng: &mut R, nodes: &mut [TestNode], elder_size: usi
 
     poll_and_resend(nodes);
 
-    expectations.verify(nodes);
+    expectations.verify(nodes, &Default::default());
```

Comment:
We might need to pass the new-to-old map here too.

Reply:
Actually no, it's fine. It seems the way this function is used is that all churn is settled by the time it is called.